frontend/src/main/pages/HomePage.js

```diff
@@ -1,32 +1,53 @@
 import BasicLayout from "main/layouts/BasicLayout/BasicLayout";
-
-
 import { useState } from "react";
 import BasicCourseSearchForm from "main/components/BasicCourseSearch/BasicCourseSearchForm";
+import { useBackendMutation } from "main/utils/useBackend";
+import { queryAllByTestId } from "@testing-library/react";
+import { toast } from "react-toastify";
 
-//temp placeholder
-/* istanbul ignore next */ 
-// Stryker disable next-line all : temporary placeholder function. This function fetches the course data given quater, level, subject
-function fetchBasicCourseJSON(event, params){
-  return null;
-}
+export default function HomePage() {
+  // Stryker disable next-line all : Can't test state because hook is internal
+  const [courseJSON, setCourse] = useState([]);
 
-//temp placeholder
-/* istanbul ignore next */ 
-// Stryker disable next-line all : temporary placeholder function. This function updates json after course list is fetched
-function setCourseJSON(json){
-  return;
-}
+  const objectToAxiosParams = (query) => ({
+    url: "/api/public/basicsearch",
+    //Removed the "method: "GET"" statement that was here because the API call still goes through even without it and mutation testing was failing for it
+    // Furthermore, although we can test whether useBackend was called or not, we can't test which method was passed in as a parameter to objectToAxiosParams (we can't test internals of useBackend)
```

Comment:
Do we need the comments in this file

Reply:
Helps explain why the "method" field isnt there but I can remove it if you feel its unecessary

Reply:
removed the second comment but kept the first